.github/workflows/trigger-deployment.yml

```
@@ -0,0 +1,24 @@
+name: Trigger deployment on tag
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  trigger-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger deployment workflow
+        run: |
+          curl -X POST \
+            -H "Accept: application/vnd.github.v3+json" \
+            -H "Authorization: token ${{ secrets.DEPLOYMENT_PAT }}" \
+            -H "Content-Type: application/json" \
+            https://api.github.com/repos/yeagerai/genlayer-studio/dispatches \
```

shouldn't this point to the infra repo? like
```suggestion
            https://api.github.com/repos/yeagerai/genlayer-simulator-infra/dispatches \
```